packages/shared/src/permissions/permissions.utils.ts

```diff
@@ -40,7 +43,8 @@ export const GLOBAL_PERMISSIONS = [
   "manageTags",
   "manageApiKeys",
   "manageIntegrations",
-  "manageWebhooks",
+  "manageEventWebhooks",
+  "manageSDKWebhooks",
```

Comment:
Thinking about this more, I think we should make a breaking change and grant `manageSDKWebhooks` permission to engineers and experimenters.  I think it was a bug to not do this originally and it will simplify the policies since we can just include this permission in the `SDKConnectionsFullAccess` policy instead of making a separate SDKWebhook one.

Reply:
Sounds good. 

So, just to clarify, we'll keep the change that removes `manageWebhooks` in lieu of the two different `manageEventWebhooks` and `manageSDKWebhooks`.

And we'll update engineer and experimenter roles to include `manageSDKWebhooks`, but we'll keep `manageEventWebhooks` as an admin-only permission.

In these changes, I think it also makes sense to change the scoping of `manageSDKWebhooks` so it's env scoped like `manageSDKConnections`.